packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -612,9 +612,12 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
           return scaledBidBook.hasOrders() || priceBook.hasOrders();
         },
         lockOraclePriceForRound() {
-          const { updatingOracleQuote } = this.state;
+          const { facets, state } = this;
+          const { updatingOracleQuote } = state;
```

Comment:
consider deeper destructuring,
```suggestion
          const { facets: { helper }, state: { updatingOracleQuote } } = this;
```
fwiw, now that you have `state`, line 619 could be shorter

Reply:
I tried the deeper destructuring, and prettier turned it into 4 lines, so I reverted.

> fwiw, now that you have state, line 619 could be shorter

Oh, yeah. Definitely an omission.